setup.cfg

```
@@ -0,0 +1,61 @@
+[metadata]
+name = gdc_client
```

To reduce confusion, make this consistent with pyproject.toml
```suggestion
name = gdc-client
```